src/concrete/ml/pytest/torch_models.py

```diff
@@ -1387,3 +1387,25 @@ def forward(self, x):
         x = torch.relu(self.fc2(x))
         x = self.fc4(self.quant_3(x))
         return x
+
+
+class PartialQATModel(torch.nn.Module):
```

Comment:
more like `HybridQatMOdel` right ?

Reply:
Well it depends, the idea here is that you can have non-qat module afterwards and only compile the `sub_module` that uses qat and everything will go fine.

Reply:
ah got it thanks !